packages/jest-worker/src/workers/utils.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import {parse as parseMessage, stringify} from 'telejson';
+
+const customMessageStarter = 'jest string - ';
+
+export function serialize(data: unknown): unknown {
+  if (data != null && typeof data === 'object') {
```

Comment:
Isn't e.g. `message` always an object? That means all `jest-worker` clients will always have to pay for the `telejson` overhead. That might be a significant penalty for larger repositories that use `jest-hastemap` or Metro where the function is called for every file in the repository/app. 

I think it would be worth to avoid this overhead by introducing a new option that allows customizing the serialization either

* by jest-worker providing a predefined set of serialisation formats (easier) 
* requiring a path to a mdoule that exports `serialize`/`deserialize.

Reply:
no, it's whatever the user passes, e.g. strings, numbers etc.

That said, I'm not against custom serializers

Reply:
I see. That is because the messages from parent to child don't use the custom serialization? I think we should make sure that the `setupArgs` in the INIT message and the arguments passed in a method call are correctly serialized as well.